### PR TITLE
Add return statement to grdclip and grdgradient docstring

### LINKED ISSUE #1388: Add returns section to grdclip docstring
The docstring in `pygmt/src/grdclip.py` needs a 'Returns' section.

To fix this issue, copy and paste L82-89 in `pygmt/src/grdcut.py`:
https://github.com/GenericMappingTools/pygmt/blob/9efe3177bb0f3774eb5d0ac0e4b3957ff3eae6e9/pygmt/src/grdcut.py#L81-L89
to add a returns section between L70 and L71 in `pygmt/src/grdclip.py`:
https://github.com/GenericMappingTools/pygmt/blob/9efe3177bb0f3774eb5d0ac0e4b3957ff3eae6e9/pygmt/src/grdclip.py#L70-L71

A local development environment is not needed to address this issue (no installation required) :tada: Just follow these instructions for editing the documentation on GitHub: https://www.pygmt.org/dev/contributing.html#editing-the-documentation-on-github.

### PR DESCRIPTION
**Pull request by kadatatlukishore**
**Description of proposed changes**

Updated the docstring in grdclip.py to have a return statement.

Fixes #1388

**Reminders**

- [x] Run `make format` and `make check` to make sure the code follows the style guide.
- [x] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [x] Write detailed docstrings for all functions/methods.
- [x] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

**Comment by kadatatlukishore** [comment]
Hey @meghanrjones , I have just added the docstring please review. Thanks.

File: pygmt/src/grdclip.py
Comment (meghanrjones) [inline pygmt/src/grdclip.py:71]:
```suggestion

    Returns
```

File: pygmt/src/grdclip.py
Comment (meghanrjones) [inline pygmt/src/grdclip.py:75]:
```suggestion

        - :class:`xarray.DataArray` if ``outgrid`` is not set
```

**Review by meghanrjones — COMMENTED** [review]
Thanks for your pull request, @kadatatlukishore! Nice work! I have made two small suggestions that add one blank line. The first is for separation from the other section of the docstring and the second is required for the list to be rendered properly in sphinx. You can check out the [Vercel preview](https://pygmt-lqohhwlvb-gmt.vercel.app/api/generated/pygmt.grdclip.html) if you want to see the built docs.

**Comment by kadatatlukishore** [comment]
Hello @meghanrjones , made the required changes. Preview look fine. Thanks for the suggestions !

**Review by meghanrjones — APPROVED** [review]
Looks great!

I'll leave this PR open for up to a day before merging, just in case any of the other maintainers have comments. Thanks again for your contribution!

**Comment by weiji14** [comment]
Seems like `grdgradient` could also use the same treatment :slightly_smiling_face: @kadatatlukishore, if you have time, do you mind copy-and-pasting the same chunk of documentation to https://github.com/GenericMappingTools/pygmt/blob/9b60f5c1be8c7412c0af3ffd847eeca8dfd7c365/pygmt/src/grdgradient.py#L94

Thanks!

**Comment by meghanrjones** [comment]
> Seems like `grdgradient` could also use the same treatment 🙂 @kadatatlukishore, if you have time, do you mind copy-and-pasting the same chunk of documentation to
> 
> https://github.com/GenericMappingTools/pygmt/blob/9b60f5c1be8c7412c0af3ffd847eeca8dfd7c365/pygmt/src/grdgradient.py#L94
> 
> Thanks!

@weiji14 do you want this change in the same or a separate pull request?

**Comment by weiji14** [comment]
> > Seems like `grdgradient` could also use the same treatment slightly_smiling_face @kadatatlukishore, if you have time, do you mind copy-and-pasting the same chunk of documentation to
> > https://github.com/GenericMappingTools/pygmt/blob/9b60f5c1be8c7412c0af3ffd847eeca8dfd7c365/pygmt/src/grdgradient.py#L94
> > 
> > Thanks!
> 
> @weiji14 do you want this change in the same or a separate pull request?

I think it's ok to have it in the same pull request since it is a related change and not too big. But ok with making it into a separate thing if it's a bit troublesome.

**Comment by kadatatlukishore** [comment]
> > > Seems like `grdgradient` could also use the same treatment slightly_smiling_face @kadatatlukishore, if you have time, do you mind copy-and-pasting the same chunk of documentation to
> > > https://github.com/GenericMappingTools/pygmt/blob/9b60f5c1be8c7412c0af3ffd847eeca8dfd7c365/pygmt/src/grdgradient.py#L94
> > > 
> > > Thanks!
> > 
> > 
> > @weiji14 do you want this change in the same or a separate pull request?
> 
> I think it's ok to have it in the same pull request since it is a related change and not too big. But ok with making it into a separate thing if it's a bit troublesome.

I can work on the same pull request. Thanks @weiji14

**Review by seisman — APPROVED** [review]

**Comment by kadatatlukishore** [comment]
Hello @meghanrjones , Should I have to make any changes ?

File: pygmt/src/grdgradient.py
Comment (seisman) [inline pygmt/src/grdgradient.py:95]:
```suggestion

```

File: pygmt/src/grdgradient.py
Comment (seisman) [inline pygmt/src/grdgradient.py:100]:
```suggestion

```

**Review by seisman — COMMENTED** [review]
Have to manually remove these whitespaces in blank lines.

**Review by willschlitzer — APPROVED** [review]

**Comment by meghanrjones** [comment]
Great work @kadatatlukishore and thanks again for addressing this issue! :tada:

**Pull request merged**
